Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferParserAVFObjC.mm

```diff
@@ -208,6 +208,13 @@ AtomString codec() const override
 
 MediaPlayerEnums::SupportsType SourceBufferParserAVFObjC::isContentTypeSupported(const ContentType& type)
 {
+    // Check that AVStreamDataParser is in a functional state.
+    if (!PAL::getAVStreamDataParserClass()) {
```

Comment:
```suggestion
    if (PAL::getAVStreamDataParserClass()) {
```

Reply:
oops, I had actually fixed that but didn't commit it, again a last minute change trying to change the code flow :(